<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>961</width>
    <height>904</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>961</width>
    <height>904</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>961</width>
    <height>904</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>941</width>
      <height>831</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QGroupBox" name="MessagesBox">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>650</height>
        </size>
       </property>
       <property name="title">
        <string>–ß–∞—Ç. –ú–∞–∫—Å–∏–º–∞–ª—å–Ω–æ–µ –∫–æ–ª-–≤–æ —Å–∏–º–≤–æ–ª–æ–≤ –≤ –æ–¥–Ω–æ–º —Å–æ–æ–±—â–µ–Ω–∏–∏ - 256. –ï—Å–ª–∏ –æ–Ω–æ –ø—Ä–µ–≤—ã—à–µ–Ω–æ - —Å–æ–æ–±—â–µ–Ω–∏–µ –±—É–¥–µ—Ç –¥—Ä–æ–±–∏—Ç—å—Å—è!</string>
       </property>
       <widget class="QScrollArea" name="MessagesScrollArea">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>20</y>
          <width>941</width>
          <height>621</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOn</enum>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>922</width>
           <height>619</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <widget class="QWidget" name="verticalLayoutWidget_2">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>941</width>
            <height>621</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="MessagesLayout">
           <property name="sizeConstraint">
            <enum>QLayout::SetFixedSize</enum>
           </property>
          </layout>
         </widget>
        </widget>
       </widget>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="ChatButtonLayout">
       <item>
        <widget class="QPushButton" name="photoButton">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>150</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>60</width>
           <height>150</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>24</pointsize>
          </font>
         </property>
         <property name="text">
          <string>üì∑</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="textEditSend">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>160</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>800</width>
           <height>180</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>16</pointsize>
          </font>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QGridLayout" name="EmojiPhotoLayout">
         <item row="1" column="0">
          <widget class="QPushButton" name="clearButton">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>60</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>25</pointsize>
            </font>
           </property>
           <property name="text">
            <string>üóëÔ∏è</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QPushButton" name="emojiButton">
           <property name="minimumSize">
            <size>
             <width>60</width>
             <height>60</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>25</pointsize>
             <stylestrategy>PreferDefault</stylestrategy>
            </font>
           </property>
           <property name="text">
            <string>üòÄ</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="sendButton">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>150</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>25</pointsize>
          </font>
         </property>
         <property name="text">
          <string>üì©</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>961</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>–ù–∞—Å—Ç—Ä–æ–π–∫–∏</string>
    </property>
    <addaction name="SetConnectionAction"/>
    <addaction name="VolumeAction"/>
    <addaction name="ClientsAction"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="SetConnectionAction">
   <property name="text">
    <string>–£—Å—Ç–∞–Ω–æ–≤–∏—Ç—å –ø–æ–¥–∫–ª—é—á–µ–Ω–∏–µ</string>
   </property>
  </action>
  <action name="About">
   <property name="text">
    <string>–û –ø–æ–¥–∫–ª—é—á–µ–Ω–∏–∏</string>
   </property>
  </action>
  <action name="VolumeAction">
   <property name="text">
    <string>–ò–∑–º–µ–Ω. –≥—Ä–æ–º–∫–æ—Å—Ç—å —É–≤–µ–¥–æ–º–ª.</string>
   </property>
  </action>
  <action name="ClientsAction">
   <property name="text">
    <string>–ö–ª–∏–µ–Ω—Ç—ã</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
